lib/proofer/version.rb

```diff
@@ -1,3 +1,3 @@
 module Proofer
-  VERSION = '1.0.0'.freeze
+  VERSION = '1.0.1'.freeze
```

Comment:
I think we should go for a 2.0.0 because the gem has changed so much since first being labeled as 1.0.0

Reply:
Done.

Reply:
Hmm. I don't agree with that. If our intent was to use semantic versioning, we should have started with a version less than 1.0, which indicates that anything may change at any time. The gem hasn't been properly working until the latest changes I made, and we're still not sure if we've worked all the kinks out. Until we do, I think we should stay at 0.x, and once we've thoroughly tested it, we can declare 1.0. We're the only ones using this gem, so I think it's fine to go back to 0.x.

Reply:
A project like this is never really "done" and I don't think we'll ever really know it's "ready enough" for a 1.0 or equivalent milestone, the numbers are arbitrary and we have done a poor job of versioning this gem at all.

I'd prefer a big number jump (`2.0.0`) because the way I see it, we're doing a true-up to more than [a year's worth](https://github.com/18F/identity-equifax-api-client-gem/commit/e2640d1e05af82381e2bf24591a08e2086b3ef32) of changes, even if they are small changes to the API.

Second to that I would accept a patch `1.0.1` because the rough API of the gem has stayed the same (even though we've introduced various breaking changes which mean it's not a good candidate for a patch)

I think going backwards `0.X` makes zero sense and would confuse our future selves.

Reply:
We haven't introduced any breaking changes because the gem itself was broken. All of these changes have been fixes, which in semantic versioning are minor version bumps. The IdP app has not worked properly with the `equifax` proofing vendor because this gem had bugs in it and because we only recently started testing with the Equifax servers. Before, we were only using a Mock vendor. 

The changes we made to this gem have enabled the IdP app to work better with each change. That's not a breaking change because it didn't require the IdP app to change, except for one change where we stopped asking for `reasons` in analytics for the financial and phone steps. That IdP change was not due to the gem making changes. It was due to us never actually testing against Equifax data.

Reply:
Well, since it was already at 1.0, then can we all agree to 1.0.1, to encompass all the prior bug fixes under one bump?

Reply:
Yes, that sounds great to me.

Reply:
Yup, I'm ok with 1.0.1

And as a point of clarification https://github.com/18F/identity-proofer-gem/pull/28/ contains a breaking change, renaming a param from `bank_acct` to `bank_account`